### PR TITLE
feat: Added xray encryption config

### LINKED ISSUE #625: Add Support for AWS::XRay::EncryptionConfig

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

**Review by disq — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -76,6 +76,8 @@ require (
 	github.com/stretchr/testify v1.7.0
 )
 
+require github.com/aws/aws-sdk-go-v2/service/xray v1.13.4 // indirect
```
Comment (bbernays) [inline go.mod:79]: Why is this an indirect dependency?

**Review by bbernays — APPROVED** [review]
LGTM... other than single comment

**Review by roneli — APPROVED** [review]
LGTM, would add the rest of the XRAY api's eventually maybe in a new issue @bbernays 

Sampling rules: https://docs.aws.amazon.com/xray/latest/api/API_GetSamplingRules.html
Groups: https://docs.aws.amazon.com/xray/latest/api/API_GetGroups.html

**Pull request merged**